tools/profile_apply

```diff
@@ -276,7 +276,12 @@ def apply(tabs):
                 section, section_rows = row['section'].strip(), []
             if row['title'].strip():
```

Comment:
Can you add a comment in the code to explain what is going on here please?

It's not clear just from the code that when a new title is found, it creates a chart divider with a dummy concept ID and the format will contain the column type and interval details.

Is it necessary to have a dummy concept ID for a section? I'm a bit uneasy about it.